tests/frontend/test_kibana_app/cypress/integration/utils/driver.js

```diff
@@ -83,18 +111,60 @@ export const updateCookies = () => {
       //   if (e.name == parameterToFilter[l]) e.value = cookie.value
       //   return e;
       cy.readFile(filename).then((obj) => {
-
         const newCookie = obj.map(e => {
         if (e.name == parameterToFilter[l]) e.value = cookie.value
         return e;
         })
         cy.writeFile(filename,  JSON.stringify(newCookie))
-
       })
     }
+   }
   });
 }
 
+// export const updateCookies2 = () => {
```

Comment:
same here, commented functions should not be in the production code.

Reply:
done